pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -716,10 +722,27 @@ func resolveBackupManifests(
 	} else {
 		// Since incremental layers were *not* explicitly specified, search for any
 		// automatically created incremental layers inside the base layer.
-		prev, err := FindPriorBackups(ctx, baseStores[0], IncludeManifest)
+
+		var incStores []cloud.ExternalStorage
+		if len(incFrom) != 0 {
+			incStores = make([]cloud.ExternalStorage, len(incFrom))
+			for i := range incFrom {
+				store, err := mkStore(ctx, incFrom[i], user)
```

Comment:
nit: ignore it if you want, for consistency to the above if condition can we assign directly to incStores[i], and then defer a close on incStores[i]?